src/main/java/io/xpring/xrpl/DefaultXpringClient.java

```diff
@@ -85,7 +87,16 @@ public BigInteger getBalance(final String xrplAccountAddress) throws XpringKitEx
      * @return The status of the given transaction.
      */
     public TransactionStatus getTransactionStatus(String transactionHash) throws XpringKitException {
```

Comment:
Per https://xrpl.org/reliable-transaction-submission.html users will typically need to know whether all of the relevant ledgers were checked. (relevant = ones that could have included the transaction, based on the latest validated ledger at the time of submission, and the transaction's LastLedgerSequence)

I don't know whether this is supported in gRPC but we should consider having rippled tell us whether the necessary set of ledgers was searched. The following PR was merged, to be included in rippled 1.5.0 - https://github.com/ripple/rippled/pull/3145

Reply:
You've asked this before :) 

It's tracked here: https://app.asana.com/0/1143722888069470/1155647690941929